tests/functional/test_lint_imports.py

```diff
@@ -102,3 +102,13 @@ def test_logging_configuration_respects_verbose_flag(verbose, capsys):
 
     # N.B. "Wrote data cache file" is logged by Grimp.
     assert ("Wrote data cache file" in captured.out) == verbose
+
+
+def test_stdout(capsys):
```

Comment:
If you look in `tests/unit/contracts` you'll see unit tests for the rendering of each of the different contracts. It's probably best to concentrate our testing there. Though I agree we should have at least one functional test to exercise the `importlib` invocation.

It does make me think we should continue with the ports-and-adapters approach we've used for other things such as the filesystem. Maybe we need an abstraction for getting the file based on the path? Let me know if you need some more pointers.